index.html

```diff
@@ -4545,28 +4545,172 @@ <h2>
+    to be valid, while invalidating old verifications that were produced with the older material.
+  </p>
+  
+  <p>
+    Rotation is a useful mechanism for protecting against verification method compromise, 
+    since frequent rotation of a verification method by the controller, 
+    reduces the value of a compromised verification method to an attacker.
+  </p>
+  
+  <p class="note">
+    Higher security environments tend to employ more frequent verification method rotation.
+  </p>
+  
+  <p class="note">
+    Frequent rotation of a verification method might be frustrating 
+    for parties that are forced to continiously renew or refresh associated credentials.
+  </p>
+  
+</section>
+
+<section>
+<h2>
+Verification Method Revocation
+</h2>
+
+<p>
+  Verification method revocation is a reactive security measure.
+</p>
+
+<p>
+  If a verification method is no longer exclusivley accessible to the 
+  controller or parties trusted to act on behalf of the controller, 
+  it should be revoked immediatly to reduce the risk of masquerading, theft and fraud.
+</p>
+
+<p class="advisement">
+  It is considered a best practice to support key revocation.
+</p>
+
+<p class="advisement">
+A controller should revoke any verification method that is believed to be compromised immediatly.
+</p>
+
+<p class="advisement">
+Cryptographic verifications associated with revoked verification methods should be considered invalid  
+regardless of when revocation occurs.
+</p>
+
+<p class="note">
+  As described in Section <a href="#verification-method-types"></a>, absence of a verification method is the only form of revocation that applies to all DID Methods.
+</p>
+
+<p>
 Section <a href="#method-operations"></a> specifies the <a>DID</a> operations to
-be supported by a <a>DID method</a> specification, including deactivation of a
-<a>DID document</a> by replacing it with an updated <a>DID document</a>. It is
-also up to the <a>DID method</a> to define how revocation of cryptographic keys
-might occur. Additionally, <a>DID method</a> specifications are also expected
-to enable support for a quorum of trusted parties to enable key recovery. Some
-of the facilities to do so are suggested in Section
-<a href="#did-controller"></a>. Not all <a>DID method</a>
+be supported by a <a>DID method</a> specification, 
+including <a href="#update">update</a> and <a href="#deactivate">deactivate</a> 
+which MAY be used to remove verification method from a DID Document.
+</p>
+
+<p>
+Even if a verification method is present in a DID Document, 
+additional information such as a public key revocation certificate,
+or external allow or deny list might be used to determine if a verification method has been revoked.
+</p>
+
+
+
+<p class="note">
+An attacker can trivially back date signatures to a period before the verification method was revoked, 
+this is the reason that revocation applies to verifications for all time, 
```

Comment:
I suggest removing this paragraph in favor of the wording I provided in a previous comment.

Reply:
I have made changes attempting to address your concern with this paragraph.